test/lib/jobFactory.test.js

```diff
@@ -298,64 +298,137 @@ describe('Job Factory', () => {
 
     describe('getPullRequestJobsForPipelineSync', () => {
         let config;
-        let returnValue;
-        let queryConfig;
+        let expectedQueryConfig;
 
         beforeEach(() => {
             sinon.stub(JobFactory.prototype, 'query').returns();
 
             config = {
-                pipelineId: '12345',
-                prNames: ['PR-2', 'PR-3']
+                pipelineId: '12345'
+            };
+        });
+
+        afterEach(() => {
+            assert.calledWith(datastore.query, expectedQueryConfig);
```

Comment:
we don't usually call assert in the afterEach hook
maybe move the assert to each test?

Reply:
Done